lib/OpenQA/Client.pm

```diff
@@ -37,7 +36,10 @@ sub new {
     }
 
     if ($args{api}) {
-        for my $file ($ENV{OPENQA_CLIENT_CONFIG}||undef, glob('~/.config/openqa/client.conf'), '/etc/openqa/client.conf') {
+        my @cfgpaths=('~/.config/openqa', '/etc/openqa');
+        @cfgpaths=($ENV{OPENQA_CONFIG},@cfgpaths) if defined $ENV{OPENQA_CONFIG};
```

Comment:
I know. I am more used to this form though, and in general find it a bit more readable and self explaining.
Can change it if you insist though...